docs/source/reference/image_classification.rst

```diff
@@ -8,7 +8,7 @@ Image Classification
 ********
 The task
 ********
-The task of identifying what is in an image is called `image classification <https://arxiv.org/abs/1912.12162>`_. Typically, Image Classification is used to identify images containing a single object. The task predicts which ‘class’ the image most likely belongs to with a degree of certainty.  A class is a label that desecribes what is in an image, such as ‘car’, ‘house’, ‘cat’ etc. For example, we can train the image classifier task on images of ants and it will learn to predict the probability that an image contains an ant.
```

Comment:
As I read the abstract of the linked paper, it's more like object detection rather than image classification, so I think this should be removed.